Formula/vmctl.rb

```diff
@@ -0,0 +1,31 @@
+class Vmctl < Formula
+  desc "Data migration tool used to migrate data from supported DBs to VictoriaMetrics"
+  homepage "https://docs.victoriametrics.com/vmctl.html"
+  url "https://github.com/VictoriaMetrics/VictoriaMetrics.git",
+      tag:      "v1.90.0",
+      revision: "b5d18c0d281b5bcd4dc13cc72d897c38fd4bb374"
+  license "Apache-2.0"
+  head "https://github.com/VictoriaMetrics/VictoriaMetrics.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "vmctl"
+    bin.install "bin/vmctl"
+  end
+
+  def caveats
+    <<~EOS
+      For more details, read:
+        https://docs.victoriametrics.com/vmctl.html
+      VictoriaMetrics on Github:
+          https://github.com/VictoriaMetrics/VictoriaMetrics
+      Feel free to ask your question in our communities:
+        https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#contacts
+    EOS
+  end
```

Comment:
```suggestion
```
We no longer add `caveats` when they are not specific to Homebrew.

Reply:
Ah, I see. I've deleted it.